src/main/resources/swagger.api/examples/authorities_ids.sample

```diff
@@ -0,0 +1,14 @@
+{
+  "authorities": [
+    {
+      "id": "66ce00f7-2a90-4919-81b3-460e04a06ae4"
+    },
+    {
+      "id": "92863cc5-b86a-4be8-aabb-2e1905e46583"
+    },
+    {
+      "id": "1baeeca2-bb23-4d7c-8bd5-0a631dd3fdd6"
+    }
+  ],
+  "totalRecords": 3
+}
```

Comment:
If it is en example to plain/text - then shouldn't it be plain format and not json?